kernel/kernel-defaults/src/test/scala/io/delta/kernel/defaults/DeltaTableReadsSuite.scala

```diff
@@ -23,8 +23,107 @@ import io.delta.golden.GoldenTableUtils.goldenTablePath
 
 import io.delta.kernel.Table
 
+import io.delta.kernel.defaults.internal.DefaultKernelUtils
+
 class DeltaTableReadsSuite extends AnyFunSuite with TestUtils {
 
+  //////////////////////////////////////////////////////////////////////////////////
```

Comment:
I think it would be good to add them in `ParquetBatchReaderSuite` as we can test just the reader without the Kernel stuff. If the existing test infra doesn't support it, I am ok to keep them here with a TODO comment.